packages/clarity-js/src/data/baseline.ts

```diff
@@ -28,12 +28,16 @@ export function track(event: Event, width: number, height: number): void {
             buffer.screenWidth = width;
             buffer.screenHeight = height;
             break;
+        case Event.Mutation:
+            buffer.activityTime = time;
```

Comment:
Let's do a separate function for activity to not mix up the function signature here.